beacon-chain/rpc/beaconv1/server.go

```diff
@@ -32,6 +32,7 @@ type Server struct {
 	ChainInfoFetcher    blockchain.ChainInfoFetcher
 	DepositFetcher      depositcache.DepositFetcher
 	BlockFetcher        powchain.POWBlockFetcher
+	PowchainInfoFetcher powchain.ChainInfoFetcher
```

Comment:
Are these changes still relevant to this PR?

Reply:
Nope. +1